fairscale/utils/testing.py

```diff
@@ -45,6 +45,14 @@
 from fairscale.nn.model_parallel import destroy_model_parallel, initialize_model_parallel
 from fairscale.nn.model_parallel.random import model_parallel_cuda_manual_seed
 
+skip_if_no_cuda = pytest.mark.skipif(
```

Comment:
this was re-implemented in pretty much every test file, not broken but small refactoring

Reply:
cool